wire-codegen-sample/build.gradle

```diff
@@ -1,4 +1,3 @@
-apply plugin: 'com.liferay.maven.plugin.builder'
```

Comment:
We should rewrite `wire-codegen-sample` to use Gradle, now it won't generate code.

Reply:
wanna tackle that in a follow-up?

Reply:
For sure!